Jenkinsfile

```diff
@@ -61,10 +61,12 @@ pipeline {
 
     stage('Check') {
       steps {
-        withCheckEnvironment {
-          gradle 'check'
+        catchError(buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
+            withCheckEnvironment {
+              gradle 'check'
+            }
+            junit allowEmptyResults: true, testResults: '**/build/test-results/test/TEST-*.xml,**/build/test-results/tests/test/TEST-*.xml,**/build/jest-reports/TEST-*.xml'
```

Comment:
I think we should collect the test results after the catch. This allows us to see the test results of the modules, even if a module fails.